### ISSUE TITLE
Font/Background issue with WLR_RENDERER=vulkan

### ISSUE THREAD

**Issue by tmdobbin** [alacritty/alacritty#8012]
I've been using alacritty on sway for a couple weeks. Today I thought I'd try out wlr's Vulkan renderer. I placed WLR_RENDERER=vulkan into /etc/environment, rebooted, and started up sway. Everything worked great, but alacritty looks very odd. The background is fully transparent (I don't have any sort of transparency set up) and I can see my desktop wallpaper through it. The font (MesloLGS NerdFont) also looks low resolution, looks like no anti-aliasing or anything. While alacritty is running, for whatever reason, commenting out the [colors.primary] key "background", then uncommenting it fixes the issue, but only until I restart alacritty.
![What it looks like on startup with WLR_RENDERER=vulkan](https://github.com/alacritty/alacritty/assets/59059995/3a3403f8-b110-417e-a1f1-5aff106a4269)
![What it looks like after commenting out the background key](https://github.com/alacritty/alacritty/assets/59059995/e0948bdc-1da3-40b3-bb2f-f2cc4d4d6ce9)
![What it looks like after uncommenting (white used intentionally as background here to show clear contrast between alacritty and desktop wallpaper, my alacritty background and wallpaper are usually very similar colours)](https://github.com/alacritty/alacritty/assets/59059995/75620b0f-6e25-44ce-b4b2-294b5e08aab9)

### System
OS: Arch Linux
Version: alacritty 0.13.2 (bb8ea18e)
Sway WM + WLR_RENDERER set to Vulkan

The logs don't seem to have anything abnormal in them.
```
Created log file at "/tmp/Alacritty-5219.log"
[0.000002717s] [INFO ] [alacritty] Welcome to Alacritty
[0.000031658s] [INFO ] [alacritty] Version 0.13.2 (bb8ea18e)
[0.000035316s] [INFO ] [alacritty] Running on Wayland
[0.000182454s] [INFO ] [alacritty] Configuration files loaded from:
                                     "/home/tmd/.config/alacritty/alacritty.toml"
[0.000393720s] [INFO ] [alacritty] winit event: NewEvents(Init)
[0.000402748s] [INFO ] [alacritty] winit event: Resumed
[0.013383386s] [INFO ] [alacritty] Using EGL 1.5
[0.016032359s] [INFO ] [alacritty] Window scale factor: 1
[0.024049979s] [INFO ] [alacritty] Running on Mesa Intel(R) Xe Graphics (TGL GT2)
[0.024067932s] [INFO ] [alacritty] OpenGL version 4.6 (Core Profile) Mesa 24.1.0-arch1.1, shader_version 4.60
[0.024072958s] [INFO ] [alacritty] Using OpenGL 3.3 renderer
[0.039290561s] [INFO ] [alacritty] Cell size: 9 x 20
[0.039315294s] [INFO ] [alacritty] Padding: 0 x 0
[0.039318001s] [INFO ] [alacritty] Width: 800, Height: 600
[0.039347088s] [INFO ] [alacritty] PTY dimensions: 30 x 88
[0.041272361s] [INFO ] [alacritty] Initialisation complete
[0.041295686s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(96999484433792)), event: Resized(PhysicalSize { width: 800, height: 600 }) }
[0.041305707s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(96999484433792)), event: RedrawRequested }
[0.042545209s] [INFO ] [alacritty] winit event: AboutToWait
[0.042570535s] [INFO ] [alacritty] winit event: NewEvents(WaitCancelled { start: Instant { tv_sec: 2048, tv_nsec: 498843609 }, requested_resume: None })
[0.042578465s] [INFO ] [alacritty] winit event: AboutToWait
[0.042936701s] [INFO ] [alacritty] winit event: NewEvents(WaitCancelled { start: Instant { tv_sec: 2048, tv_nsec: 498868672 }, requested_resume: None })
[0.042944327s] [INFO ] [alacritty] winit event: WindowEvent { window_id: WindowId(WindowId(96999484433792)), event: Resized(PhysicalSize { width: 638, height: 1048 }) }```

**Comment by chrisduerr** [comment]
Clearly not an Alacritty issue.

**Comment by tmdobbin** [comment]
> Clearly not an Alacritty issue.

How so? Every other app works properly other than Alacritty which does work if I essentially foece it to reload the config. Who should I submit an issue to instead?

**Comment by tmdobbin** [comment]
In case anyone else happens upon this issue, the wlroots devs have helped identify that the master branch of sway fixes this issue (1.10-dev-2e9139df at time of writing)
